app/routes/brand/components/BrandPage.css

```diff
@@ -1,3 +1,7 @@
 .root {
-  color: red;
+  color: #777;
```

Comment:
Preferable use global css vars from `app/styles/variables.css` and add new one as needed.